src/tracing/trace_event.h

```diff
@@ -0,0 +1,1710 @@
+// Copyright 2015 the V8 project authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SRC_TRACING_TRACE_EVENT_H_
+#define SRC_TRACING_TRACE_EVENT_H_
+
+#include <stddef.h>
+
+#include "v8-platform.h"
```

Comment:
What specific parts of this file require v8-platform?  Needing this might imply that the API is tied to v8 in some way.